### ISSUE TITLE
`--extended` flag in `config show` command with `--find` and `--label` flags.

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#20]
In analogy with [this](https://github.com/slimcoin-project/pacli/issues/18#issuecomment-1937657353).
Wouldn't it make sense to suppress the `--extended` flag usage for the following commands:
`config show VALUE CATEGORY -e/extended -f/--find`
`config show VALUE CATEGORY -e/extended -l/--label` ?

**Comment by d5000** [comment]
Yes, it would (see #18) , but again, I think it's better the other way around (suppress -c). I think this one can be closed.

**Comment by buhtignew** [comment]
I was able to use both `config show f4e26f3a14a7fbe7ea45bd60ad9005ebbaa36a3cecd864c7f3ef517c5fe27c86 transaction  -f` and `config show f4e26f3a14a7fbe7ea45bd60ad9005ebbaa36a3cecd864c7f3ef517c5fe27c86 -e transaction  -f` thus maybe we should put `-e/--extended` flag mention in the help into the square brackets to say that the flag is optional in this case?

**Comment by d5000** [comment]
In theory `-e` should now never be optional if you want to deal with the extended config file. There should be an error message ("Error: Category does not exist or is missing. etc.") shown. That happens if I introduce your second variant, while in the first one I get an empty list, but that's because I didn't store that transaction.

Can you try again with the newest code (Feb 13)? It's possible the fix I applied for the `checkpoint set` command fixed this problem too (commit 1744b71).

**Comment by buhtignew** [comment]
> Can you try again with the newest code (Feb 13)? It's possible the fix I applied for the checkpoint set command fixed this problem too (commit https://github.com/slimcoin-project/pacli/commit/1744b718362b94a0d57f7362b4b786b46fdb6d3d).

What command should I test in this case?

**Comment by d5000** [comment]
The idea was `config show f4e26f3a14a7fbe7ea45bd60ad9005ebbaa36a3cecd864c7f3ef517c5fe27c86 transaction  -f` to see if the error message is thrown but I saw it hasn't solved the "problem". It is still possible to use the argument without the flag keyword (i.e. as a posiitional keyword).

I think there's no way other to clarify that in the help file.

**Comment by buhtignew** [comment]
I think this issue is to be closed.
